index.html

```diff
@@ -257,10 +264,16 @@ <h2 class="contact-description dark-blue">
             class="change"
             required
           ></textarea>
-          <img src="img/contact-form-png.svg" alt="just background png" />
+          <img
```

Comment:
- [ ]  About section is almost ready, just try placing the items under the title in one or two lines, but not in three, like in the Figma template:

Template:
![Captura de pantalla 2022-03-25 a la(s) 11 07 11 p  m](https://user-images.githubusercontent.com/83717285/160224225-f0d2ceaf-d011-4da3-9705-8a88b0e1a062.png)

Your design:
![Captura de pantalla 2022-03-25 a la(s) 11 07 25 p  m](https://user-images.githubusercontent.com/83717285/160224228-804ba9f6-ab3b-4f4f-a001-d51ee3c43776.png)

Reply:
Done, thank you for the observation.

Reply:
resolved 👍